### PR TITLE
Modified Script for using templates and Updated Readme

### LINKED ISSUE #10: Add install script
Make this an npm script(s) to allow for easy installation (similar to npx create-react-app). The basic command `npx create-men-template` that installs the basic template with the Ejs view engine. Additional flags, like `--react` for example may be added to allow for additional frameworks or tools to be added.

### LINKED ISSUE #13: Create a Readme
A readme file is needed to show how to use this CLI. It should have instructions on how to run the basic command as well as the commands for creating a template with a framework.

### PR DESCRIPTION
**Pull request by HaiderSultanArc**
Hey, I have modified the script so that we can now use Templates like --react to create react template project using command npx create-mern-template project-name --react. The default command npx create-mern-template project-name will create the project using the template provided in the template folder. I also updated Readme.

### CODE REVIEW

**Comment by cdthomp1** [comment]
Thanks for taking the time and working on this, I will checkout the code and get back to you soon!

File: readme.md [hunk 1]
```diff
@@ -1 +1,14 @@
 # Mongo Express (React, Ejs) Node Template
+
+## Usage:
+### Basic Command:
+
+npx create-mern-template project-name
```
Comment (cdthomp1) [inline readme.md]: I would wrap these in back ticks, to give this effect `npx create-mern-template project-name`

File: readme.md [hunk 2]
```diff
@@ -1 +1,14 @@
...
+
+Use this command to create your mern app, replace project-name with your project's name. After running above command use cd project-name to navigate into your project and run npm install to install the required packages.
+
+### Using Templates:
+
+To use template use following command:
+
+npx create-mern-template project-name --react
```
Comment (cdthomp1) [inline readme.md]: Back ticks here too :)

**Comment by cdthomp1** [comment]
@HaiderSultanArc I have added just a couple of small comments. Everything looks good to go after those are done! Great work!

File: readme.md (hunk 2 above)
Comment (HaiderSultanArc) [reply readme.md]: Yes, I can format it better. Thanks for the suggestion

**Comment by cdthomp1** [comment]
Awesome job!

**Pull request merged**